File: Leetcode/Leet0832.java
//https://leetcode.com/problems/flipping-an-image/
class Solution {
    public int[][] flipAndInvertImage(int[][] image) {
        int[][] arr = new int[image.length][image.length];
        for(int row=0; row < image.length; row++){
            for(int col=0; col < image[row].length; col++){
            arr[row][col] = image[row][image[row].length - 1 - col]; //reverse
            arr[row][col] = arr[row][col] == 1?0:1; //invert
            }
        }return arr;
    }
}

File: Leetcode/Leet0832.py
class Solution:
    def flipAndInvertImage(self, image: List[List[int]]) -> List[List[int]]:
        arr = list()
        for row in image:
            arr.append(row[::-1])
            for i in range(len(arr[-1])):
                if arr[-1][i] == 0:
                    arr[-1][i] = 1
                else:
                    arr[-1][i] = 0
        return arr
        
        # for i in range(len(image)):
        #     image[i] = image[i][::-1]
        #     for j in range(len(image[i])):
        #         if image[i][j] == 0:
        #             image[i][j] = 1
        #         else:
        #             image[i][j] = 0
        # return image
            
        